packages/scaffolding-cli/src/domain/config/static.config.json

```diff
@@ -321,5 +347,75 @@
         "dest": "docs"
       }
     ]
+  },
+  "gkeInfra": {
```

Comment:
naming include devops?

Reply:
ah yeah ... no though we should refactor maybe... 
if deployment tool name is omitted it's TFS else it's jenkins etc...